addons/purchase/migrations/11.0.1.2/pre-migration.py

```diff
@@ -6,6 +6,24 @@
 from openupgradelib import openupgrade
 
 
+def detect_product_by_supplier(cr):
+    cr.execute(
+        """
+        SELECT id
+        FROM ir_model_data
+        WHERE module = 'product_by_supplier' AND
```

Comment:
This is going to be always Falsy, as you have merged module in `purchase`. Any way, do this or other check in post for avoiding this ugly hack of creating a fake column. Do you know that in OpenUpgrade views are not removed? Use that.

Reply:
Thanks for the complementary information, Stefan!